src/services/TeamService.js

```diff
@@ -125,21 +137,6 @@ async function getTeamDetail (currentUser, projects, isSearch = true) {
           if (findMember && findMember.photoURL) {
             item.photo_url = findMember.photoURL
           }
-
-          if (!isSearch) {
-            // If call function is not search, add job field
-            const findRbs = _.find(rbs, { userId: item.id })
-            if (findRbs) {
-              item.customerRate = findRbs.customerRate
-              const job = _.find(jobs, { id: findRbs.jobId })
-              if (job) {
-                item.job = {
-                  id: job.id,
-                  name: job.description
-                }
-              }
-            }
-          }
```

Comment:
Just to confirm @imcaizheng before I didn't see this returned for `resources` were there some kind of issue?

Reply:
The resources field is attached to res at https://github.com/topcoder-platform/taas-apis/pull/38/files/e192960abae4473304a1b5cd92820d36578c3de5#diff-821570075559f59a1fc7d0ac9e41b013516a19e4d7a2e81c76d3b5168589c2d3R129

Reply:
I tested locally. The jobIds in resources are populated correctly.